components/odh-notebook-controller/controllers/notebook_webhook.go

```diff
@@ -691,8 +691,7 @@ func SetContainerImageFromRegistry(ctx context.Context, config *rest.Config, not
 									tagMap := t.(map[string]interface{})
 									tagName := tagMap["tag"].(string)
 									if tagName == imageSelected[1] {
-										items := tagMap["items"].([]interface{})
-										if len(items) > 0 {
+										if items, ok := tagMap["items"].([]interface{}); ok && items != nil && len(items) > 0 {
```

Comment:
I do know that

```
items != nil && len(items) > 0
```

is overkill, but I want to have a smooth review, so explicitly mentioning the cases